js/hoot/model/rest.js

```diff
@@ -393,6 +394,25 @@ Hoot.model.REST = function (command, data, callback, option) {
 
                     JobStatusStopTimer(resp);
                 }
+
+                if (resp.status === 'complete') {
+                    var prettyString = function(obj) {
+                        var finalString = '';
+                        for (var atr in obj) {
+                            // if (atr === 'commandDetail') {
```

Comment:
You're right, that's likely going in the modal window now, or a similar function. I'll take it out.